src/NumberInput/NumberInput.svelte

```diff
@@ -99,12 +101,12 @@
   /** Obtain a reference to the input HTML element */
   export let ref = null;
 
-  import { createEventDispatcher } from "svelte";
-  import Add from "../icons/Add.svelte";
-  import Subtract from "../icons/Subtract.svelte";
-  import WarningFilled from "../icons/WarningFilled.svelte";
-  import WarningAltFilled from "../icons/WarningAltFilled.svelte";
-  import EditOff from "../icons/EditOff.svelte";
+  import { createEventDispatcher, tick } from "svelte";
+  import Add from "carbon-components-svelte/src/icons/Add.svelte";
+  import Subtract from "carbon-components-svelte/src/icons/Subtract.svelte";
+  import WarningFilled from "carbon-components-svelte/src/icons/WarningFilled.svelte";
+  import WarningAltFilled from "carbon-components-svelte/src/icons/WarningAltFilled.svelte";
+  import EditOff from "carbon-components-svelte/src/icons/EditOff.svelte";
```

Comment:
Don't see a compelling reason to change the imports here, it's not like this is going up five levels.

Reply:
Thats's fine. I forgot to change it back. This is a copy and paste from the code in our custom component mentioned in the issue, so the locations are different. Fixing.